drivers/hwmon/ina2xx.c

```diff
@@ -213,7 +213,7 @@ static const struct ina2xx_config ina2xx_config[] = {
  * table 3 AVG bit settings, found at
  * https://www.ti.com/lit/ds/symlink/ina226.pdf.
  */
-static const int ina226_avg_tab[] = { 1, 4, 16, 64, 128, 256, 512, 1024 };
+static const s32 ina226_avg_tab[] = { 1, 4, 16, 64, 128, 256, 512, 1024 };
```

Comment:
these `int` -> `s32` conversions are incomplete;
i.e.: there are still places where `int` is still used for the array of `s32` values;
either all places get converted from `int` -> `s32` or we remove them from the current patches;

but `u16` -> `s32` or `u32` -> `s32` are mandatory now (since the compiler should complain about incompatible pointer types);